src/program/snabb_lwaftr/tests/data/binding.table

```diff
@@ -1,7 +1,13 @@
 {
- {'127:2:3:4:5:6:7:128', '178.79.150.233', 1, 100, '8:9:a:b:c:d:e:f'},
- {'127:11:12:13:14:15:16:128', '178.79.150.233', 101, 64000},
- {'127:22:33:44:55:66:77:128', '178.79.150.15', 5, 7000},
- {'127:24:35:46:57:68:79:128', '178.79.150.2', 7800, 7900, '1E:1:1:1:1:1:1:af'},
- {'127:14:25:36:47:58:69:128', '178.79.150.3', 4000, 5050, '1E:2:2:2:2:2:2:af'}
+ {'127:2:3:4:5:6:7:128', '178.79.150.233', {80, 0, 16}, '8:9:a:b:c:d:e:f'},
+ {'127:11:12:13:14:15:16:128', '178.79.150.233', {2300, 0, 16}},
```

Comment:
I don't want to call them 'shift' and 'length'; they are all lengths to me. I agree that a/k are overly cryptic.
Did we decide on forcing a = 0 and contiguous port ranges or not? If we did, it doesn't need a name.
k could reasonably be replaced with psid_len, I think.